server/src/immich/api-v1/asset/asset-repository.ts

```diff
@@ -109,7 +109,9 @@ export class AssetRepository implements IAssetRepository {
         faces: {
           person: true,
         },
-        stack: true,
+        stack: {
+          exifInfo: true,
```

Comment:
For this need other relations? Should it essentially match assets? So exif, people, etc?

Reply:
Hmm, I think you are right. But to be 100% sure, I think the necessity of each relation should be examined

Reply:
Do people work before/after stacking and unstacking an asset? I assume if stacked, and then refresh the screen, and then unstack and view detail people would not show up unless they are lazy loaded in the asset-viewer itself.

Reply:
I see no issues with `people`. The `DetailPanel` component invokes the `getAssetById` function for the asset, including the case when we click the thumbnail of the child in the stack
So it seems we don't need to add faces/person relation here now. However, we definitely should remember that child assets returned by `getAssetById` contain less information comparent to the parent asset